doc/source/getting_started/index.rst

```diff
@@ -8,17 +8,31 @@ Getting started
 .. toctree::
    :maxdepth: 1
    :hidden:
+   :caption: Introduction
 
    project
    install_mapdl
    install_pymapdl
    launcher
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: Getting started
```

Comment:
```suggestion
   :caption: Resources
```
The title and caption are currently the same, which may cause confusion. How about changing 'Getting Started' to 'Learning Resources' or 'resources'?

Reply:
Good idea! Thank you!